### ISSUE TITLE
Plugin crash for several weeks

### ISSUE THREAD

**Issue by labsektor** [FriendsOfGalaxy/galaxy-integration-steam#153]
The plugin has not been working for several weeks.
Steam cannot be integrated with Gog.
Reinstalling the plugin doesn't help.
[logs.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/9192783/logs.zip)

**Comment by PescheHelfer** [comment]
Yeah, it seems like the main developer is no longer working on the project. The last release is from last November. So if nobody else jumps in, GOG Galaxy has a huge problem. I don't understand why GOG themselves do not support the development of this plugin. If they loose Steam integration, GOG Galaxy is done (which would be a huge shame) :(

**Comment by balbertho** [comment]
Seems that %localappdata%\GOG.com\Galaxy\plugins\installed\steam_plugin_config.ini is garbled, I tried to rename the file and setup the connection again in GOG Galaxy and it seems working again.

**Comment by labsektor** [comment]
Thanks Albert. It helps.

**Comment by balbertho** [comment]
@labsektor You're welcome. Seems similar issues appear in #147 and #138, so this issue appears long times ago.
